locale/en_US/locale.po

```diff
@@ -191,6 +191,15 @@ msgstr "This format will be available at a separate website"
 msgid "grid.catalogEntry.remoteURL"
 msgstr "URL of remotely-hosted content"
 
+msgid "grid.catalogEntry.isbn"
+msgstr "ISBN"
+
+msgid "grid.catalogEntry.isbn13.description"
+msgstr "ISBN 13, for example 978-951-98548-9-2"
+
+msgid "grid.catalogEntry.isbn10.description"
+msgstr "ISBN 10, for example 951-98548-9-4"
```

Comment:
> A 13-digit ISBN code, such as 978-951-98548-9-2.

> A 10-digit ISBN code, such as 951-98548-9-4.

Are these testing or demo ISBNs? I want to make sure we're not using a valid ISBN that may be in use somewhere.

Reply:
These are from Wikipedia: https://fi.wikipedia.org/wiki/ISBN
I think they are demo ISBNs issued by the Finnish national library.

Reply:
Sorry, I wasn't clear, but can you change the text in the `msgstr` for both cases to the text I posted in my last comment?

Reply:
sure

Reply:
Sorry to be picky, but please use the sentences as I wrote them:

> A 13-digit ISBN code, such as 978-951-98548-9-2.

> A 10-digit ISBN code, such as 951-98548-9-4.

Reply:
check